ds/data/processed/industry/location_quotient_high_tech.nuts2.yaml

```diff
@@ -0,0 +1,27 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: POST
+data_date: 20210602
+title: Location quotient of employment in high tech industries
+subtitle: This indicator measures the location quotient of employment in high tech industries based on the Business Register Employment Survey. High tech industries are defined in terms of SIC-4 (industry) codes, using the definition in "The Geography of the UK's creative and high-tech economies" (https://media.nesta.org.uk/documents/geography_uks_creative_high-tech_economieswv20151.pdf) in table 6.1.3.4.  
+endpoint_url: http://www.nomisweb.co.uk/api/v01/dataset/
+framework_group: place_potential
+is_experimental: True
+order: [year, nuts_id, nuts_year_spec, value.id]
+region:
+  type: NutsRegion
+  level: 2
+  source: PROVIDED
+schema:
+  region_id:
+    type: NutsRegion.nuts_id
+  region_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    id: location_quotient_high_tech
+    label: Location quotient of employment in high tech industries
+    data_type: int
```

Comment:
I think this needs to be a `float`, and we wneed to add `format: .2f` (assuming  2 decimals are enough), otherwise we'd see something like this:

<img width="499" alt="Screenshot 2021-06-02 at 10 38 58" src="https://user-images.githubusercontent.com/1309648/120458111-ea621180-c38e-11eb-90d5-6ca62c6a794c.png">

<img width="466" alt="Screenshot 2021-06-02 at 10 39 10" src="https://user-images.githubusercontent.com/1309648/120458117-ec2bd500-c38e-11eb-9202-8dae99a5c50a.png">

Reply:
Thanks @mindrones, I have made these changes.